lexicons/com/atproto/admin/getModerationReports.json

```diff
@@ -10,6 +10,11 @@
         "properties": {
           "subject": { "type": "string" },
           "ignoreSubjects": { "type": "array", "items": { "type": "string" } },
+          "actionedBy": {
+            "type": "string",
+            "format": "did",
+            "description": "Get all reports that were actioned by a specific moderator"
+          },
```

Comment:
Think there's any chance we may want this to be an array in the future?  I don't have a strong opinion on it, but if it seems likely to you, now might be a good time to preemptively accept a list of dids.

Reply:
I thought about it and decided that it probably won't be a use-case any time soon. Can't think of cases where we would want to see reports actioned by multiple folks but LMK if I'm missing anything obvious there.